src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/AbstractImportControl.java

```diff
@@ -98,15 +98,18 @@ else if (parent == null) {
                 result = AccessResult.ALLOWED;
             }
             else {
-                result = AccessResult.DISALLOWED;
+                result = AccessResult.FORBIDDEN;
             }
         }
         else {
             if (strategyOnMismatch == MismatchStrategy.ALLOWED) {
                 result = AccessResult.ALLOWED;
             }
-            else if (strategyOnMismatch == MismatchStrategy.DISALLOWED) {
-                result = AccessResult.DISALLOWED;
+            else if (strategyOnMismatch == MismatchStrategy.FORBIDDEN
+            // how to test this?
+            // || strategyOnMismatch == MismatchStrategy.DISALLOWED
+            ) {
```

Comment:
how to test this ?
we need this case to be downwards compatible, right ?

Old syntax would be allowed by schema, but not picked up and applied.

Reply:
its approved.